docs/migration.md

```diff
@@ -60,6 +60,8 @@ consider additional positioning prop support on a case-by-case basis.
 
 #### @zendeskgarden/react-dropdowns
 
+- The `v8` version of this package is no longer maintained and is
+  renamed to `@zendeskgarden/react-dropdowns.legacy` in `v9`
```

Comment:
Open to ideas on if a better location in this file or clearer description would be useful.

Reply:
Seems as good as anything to me.